runtime-common/orocos-kdl/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=orocos-kdl
+PKGSEC=libs
+PKGDEP="catkin-pkg eigen-3 boost cppunit"
```

Comment:
Why is `cppunit` a runtime dependency (and also for eigen-3)? Are they referenced anywhere or called by any program features?